app/src/main/res/layout/item_marker.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/titleText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textAlignment="center"
+        android:textSize="18sp"
+        tools:text="Naruto book" />
```

Comment:
What is that?

Reply:
This is the XML layout of the marker used in the map
<img width="74" alt="Screenshot 2021-04-22 at 14 59 35" src="https://user-images.githubusercontent.com/1939395/115718390-5c503f80-a37b-11eb-9f5e-a1c187ba0d65.png">

Reply:
Yes but why the "naruto book"?

Reply:
every attribute that begins with `tools:` is only visible in Android Studio when creating the layout
So it's just a placeholder value to avoid having blank TextViews 
<img width="254" alt="Screenshot 2021-04-22 at 15 24 56" src="https://user-images.githubusercontent.com/1939395/115722056-e6e66e00-a37e-11eb-8811-99ee4e21cb19.png">